waspc/test/Generator/WriteFileDraftsTest.hs

```diff
@@ -53,7 +53,9 @@ spec_WriteFileDrafts =
                   updatedFdsWithChecksums,
                 deletedRelPathsToChecksums
               ]
-      fileDraftsToWriteAndFilesToDelete (Just relPathsToChecksums) fdsWithChecksums `shouldBe` (map fst $ newFdsWithChecksums ++ updatedFdsWithChecksums, map fst deletedRelPathsToChecksums)
+      let (fileDraftsToWrite, filesToDelete) = fileDraftsToWriteAndFilesToDelete (Just relPathsToChecksums) fdsWithChecksums
+      fileDraftsToWrite `shouldMatchList` map fst (newFdsWithChecksums ++ updatedFdsWithChecksums)
+      filesToDelete `shouldMatchList` map fst deletedRelPathsToChecksums
```

Comment:
`shouldMatchList` does a list comparison, but does not care about ordering.

Reply:
Cool, didn't know about it!